pages/linux/ss.md

```diff
@@ -2,26 +2,26 @@
 
 > Utility to investigate sockets.
 
-- Dump all TCP/UDP/RAW/UNIX sockets:
+- Show all TCP/UDP/RAW/UNIX sockets:
 
 `ss -a {{-t/-u/-w/-x}}`
```

Comment:
We need to decide if we are to allow this, what character to use (`/`, `!`, `,`), and where this mechanism is admissible. In terms of formatting, I'd suggest we adhere to the [docopt](http://docopt.org/) standard as much as possible, i.e. use `|` for separating mutually exclusive options.